Features/Stories/FieldTypes/TextlineFieldType.feature

```diff
@@ -0,0 +1,246 @@
+Feature: Test the validations done on fields from PlatformUI - text line fieldtype
+    In order to validate the text line fieldtype
+    As an Editor user
+    I need to be able to create and update content with text line fieldtypes
+
+    Background:
+       Given I am logged in as an 'Administrator' in PlatformUI
+
+    ##
+    # Validate the existence of expected fields from a field type when creating a content
+    ##
+    @javascript @common
+    Scenario: A Content of an Content Type that has a text line fieldtype must have a text field
+        Given a Content Type with an "text line" Field exists
+        When I create a content of this type
+        Then I should see a "text line" field
+
+    @javascript @common
+    Scenario: When editing  Content, the label of a text line field must have the same name than field type from the respective Content Type
+        Given a Content Type with an "text line" with field definition name "Test text" exists
+        When I create a content of this type
+        Then I should see a "Test text" label related with the "text line" field
+
+    @javascript @common
+    Scenario: The label of a required text line field of a Content must be marked as required
+        Given a Content Type with a required "text line" with field definition name "Required" exists
+        When I create a content of this type
+        Then the "Required" field should be marked as required
+
+    ##
+    # Creating Content using a Content Type that has a text line Field Type
+    ##
+    @javascript @common
+    Scenario: Creating a text line Field works
```

Comment:
This is kind of wrong and/or inexact.
`Scenario: A Content item with a Text Line Field can be published` maybe ?